index.d.ts

```diff
@@ -11,9 +11,7 @@ declare namespace Spinnies {
     const dots: Spinner;
     const dashes: Spinner;
 
-    type Color =
-      ChalkColor
-      | "none";
+    type Color = typeof ChalkColor | "none";
```

Comment:
I don't think that this change is correct, sorry. `ChalkColor` is an alias of type `Color` exported by chalk's types.

Chalk [defines `Color`](https://github.com/chalk/chalk/blob/v4.1.2/index.d.ts#L56) as a union of types [`BackgroundColor`](https://github.com/chalk/chalk/blob/v4.1.2/index.d.ts#L6-L24) and [`ForegroundColor`](https://github.com/chalk/chalk/blob/v4.1.2/index.d.ts#L31-L49) which are themselves just unions of string literal types.

I think by taking `typeof ChalkColor | "none"` here, you're giving `Color` a type of `string | "none"`, which should be equivalent to just `"string"`. By leaving it as-is, the type system should be validating that the string literals passed to chalk are correct when choosing a color.

Reply:
Also note the code I linked above for the definition of type `Color` in chalk's source. Notice that it's defined as `type Color = ForegroundColor | BackgroundColor` not `type Color = typeof ForegroundColor | typeof BackgroundColor`. 

I'm using the same syntax here, so if the code here is incorrect, the typings for chalk are also incorrect.

Reply:
Hey thanks for the reply, I was importing it as a module
```typescript
import Spinnies from "@trufflesuite/spinnies";

const spinnies = new Spinnies();
```
and it will throw an error
`node_modules/@trufflesuite/spinnies/index.d.ts:15:7 - error TS2749: 'ChalkColor' refers to a value, but is being used as a type here. Did you mean 'typeof ChalkColor'?`

My project is full esm, configured `package.json` `type: "module"` and so the `tsconfig.json` with `nodenext`

I tried to reproduce the error by installing it on a fresh build and still showing the type error. here's the only packages and versions installed

- node: ^18
- typescript: ^4.7.4
- chalk: ^5.0.1
- and this package

![image](https://user-images.githubusercontent.com/25083033/182591525-a71fe9a1-984c-4edf-85f9-29a648747096.png)

after I updated the type manually it works fine

Reply:
I don't see how you're getting that error, as `ChalkColor` is definitely a type and not a value. Is something else in your project defining a class, variable, or function called `ChalkColor`?

If you could share a link to your project I'll have a look for myself tomorrow. This does seem quite strange.

Reply:
here I created the example you can check https://github.com/davidkhierl/trufflesuite-spinnies-example

Reply:
Thanks so much for doing that.

This change _is_ correct, and I managed to figure out what the issue is.

The problem here is that chalk is a CommonJS module and it's using the `exports = { ... }` TypeScript syntax for its main export. This means that even though the `Color` type from that module is a "pure" type (union of string literals), it is being treated as a value by the TypeScript compiler.

TIL!